src/kili/use_cases/question/question_use_case.py

```diff
@@ -0,0 +1,60 @@
+"""Question use cases."""
+
+from dataclasses import dataclass
+from typing import List, Optional, cast
+
+from kili.adapters.kili_api_gateway.issue.types import IssueToCreateKiliAPIGatewayInput
+from kili.domain.asset.asset import AssetExternalId, AssetId
+from kili.domain.project import ProjectId
+from kili.domain.question import QuestionId
+from kili.use_cases.base import BaseUseCases
+from kili.use_cases.utils import UseCasesUtils
+
+
+@dataclass
+class QuestionToCreateUseCaseInput:
+    """Question to create use case input."""
+
+    text: Optional[str]
+    asset_id: Optional[AssetId]
+    asset_external_id: Optional[AssetExternalId]
```

Comment:
I think @theodu put that kind of dataclass in a types.py module (cf Issues use cases)

Reply:
since it is always imported with the question use case, no need to put it into a separate file